### ISSUE TITLE
Error on mexing the file

### ISSUE THREAD

**Issue by dipu641298** [AlanLuSun/High-quality-ellipse-detection#11]
Hi I am getting this error can anyone tell me what is the issue ?

Error using mex
C:\opencv\build\x64\vc11\lib\opencv_core249.lib: error adding symbols: File in wrong format
collect2.exe: error: ld returned 1 exit status

**I ran this command :**

mex generateEllipseCandidates.cpp -IC:\opencv\build\include -IC:\opencv\build\include\opencv -IC:\opencv\build\include\opencv2 -LC:\opencv\build\x64\vc11\lib -I'C:\Program Files\Polyspace\R2019a\extern\include' -L'C:\Program Files\Polyspace\R2019a\extern\lib\win64\microsoft' -lopencv_core249 -lopencv_highgui249 -lopencv_imgproc249 -llibmwlapack.lib

**Comment by libbyzx** [comment]
> Hi I am getting this error can anyone tell me what is the issue ?
> 
> Error using mex
> C:\opencv\build\x64\vc11\lib\opencv_core249.lib: error adding symbols: File in wrong format
> collect2.exe: error: ld returned 1 exit status
> 
> **I ran this command :**
> 
> mex generateEllipseCandidates.cpp -IC:\opencv\build\include -IC:\opencv\build\include\opencv -IC:\opencv\build\include\opencv2 -LC:\opencv\build\x64\vc11\lib -I'C:\Program Files\Polyspace\R2019a\extern\include' -L'C:\Program Files\Polyspace\R2019a\extern\lib\win64\microsoft' -lopencv_core249 -lopencv_highgui249 -lopencv_imgproc249 -llibmwlapack.lib

I meet the same problem. Has this problem been solved?

**Comment by dipu641298** [comment]
No..!! Till now I haven't found any solution. I will let you know if I find any. And I hope you'll let me know too if you find any solution.

**Comment by dipu641298** [comment]
> No..!! Till now I haven't found any solution. I will let you know if I find any. And I hope you'll let me know too if you find any solution.

**Comment by dipu641298** [comment]
> > Hi I am getting this error can anyone tell me what is the issue ?
> > Error using mex
> > C:\opencv\build\x64\vc11\lib\opencv_core249.lib: error adding symbols: File in wrong format
> > collect2.exe: error: ld returned 1 exit status
> > **I ran this command :**
> > mex generateEllipseCandidates.cpp -IC:\opencv\build\include -IC:\opencv\build\include\opencv -IC:\opencv\build\include\opencv2 -LC:\opencv\build\x64\vc11\lib -I'C:\Program Files\Polyspace\R2019a\extern\include' -L'C:\Program Files\Polyspace\R2019a\extern\lib\win64\microsoft' -lopencv_core249 -lopencv_highgui249 -lopencv_imgproc249 -llibmwlapack.lib
> 
> I meet the same problem. Has this problem been solved?

No..!! Till now I haven't found any solution. I will let you know if I find any. And I hope you'll let me know too if you find any solution.

**Comment by libbyzx** [comment]
OK! No problem.
Thank you very much.

**Comment by xinzhengyi** [comment]
一样的错误! 我在尝试这个方案后解决了!!https://blog.csdn.net/Crystal_colouro/article/details/82960660

**Comment by real-lixin** [comment]
@xinzhengyi 
还需要下载VS软件？我安装的是Matlab2017和mingw64，mex失败了
